app/models/project.rb

```diff
@@ -6,6 +6,22 @@ class Project < ApplicationRecord
 
   validates :title, presence: true
   validates :info, presence: true
+  validates :deadline, presence: true
+  validates :cost_type, presence: true
+  validates :project_type, presence: true
+  validate :valid_deadline_date?
 
   include Searchable::Project
+
+  COST_TYPE = { price: 0, contract_price: 1, free: 2 }.freeze
+  enum cost_type: COST_TYPE
+
+  TYPE_TASK = { project_type: 0, test_type: 1 }.freeze
+  enum project_type: TYPE_TASK
+
+  protected
+
+  def valid_deadline_date?
```

Comment:
Rename to validate_deadline_date, method with quesrltionmark usually return Boolean and don't mutate state

Reply:
rename